docs/introduction/Car-registry-tutorial.rst

```diff
@@ -216,7 +221,7 @@ Few comments about implementation:
             );
         }
 
-    2. ``CarBox`` defines his own unique id by implementation of the function ``public byte boxTypeId()``. Similar function is defined in ``CarBoxData`` but it is a different ids despite value returned in ``CarBox`` and ``CarBoxData`` is the same.
+    2. ``CarBox`` defines its own unique id by implementation of the function ``public byte boxTypeId()``. A similar function is defined in ``CarBoxData`` but it is a different id despite the value returned in ``CarBox`` and ``CarBoxData`` being the same.
```

Comment:
@zennermac 

    2. A ``CarBox`` defines its own unique id by implementing the function ``public byte boxTypeId()``. A similar function is defined in ``CarBoxData`` but it is a different id despite the value returned in ``CarBox`` and ``CarBoxData`` being the same.